lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -2672,7 +2672,10 @@ void __cdecl _alloca_probe_16()
             {
                 JavascriptFunction* exceptionFunction = unhandledExceptionObject->GetFunction();
                 // This is for getcaller in window.onError. The behavior is different in different browsers
-                if (exceptionFunction && scriptContext == exceptionFunction->GetScriptContext())
+                if (exceptionFunction 
+                    && scriptContext == exceptionFunction->GetScriptContext()
+                    && exceptionFunction->IsScriptFunction()
+                    && !exceptionFunction->GetFunctionBody()->GetIsGlobalFunc())
                 {
                     *value = exceptionFunction;
```

Comment:
Can we assign the value to null on top of this function. GetCallerProperty is called from GetPropertyBuiltIns which is used in JavascriptFunction::GetProperty without checking the return value.